ui/js/component/app/view.jsx

```diff
@@ -11,12 +11,18 @@ class App extends React.PureComponent {
       checkUpgradeAvailable,
       updateBalance,
       fetchRewardedContent,
+      getThemes,
+      setTheme,
     } = this.props;
 
     document.addEventListener("unhandledError", event => {
       alertError(event.detail);
     });
 
+    getThemes();
+
+    setTheme();
```

Comment:
it has a fallback to do that: https://github.com/btzr-io/lbry-app/blob/master/ui/js/actions/settings.js#L55